src/_includes/templates/forms/file-a-privacy-complaint-form.html

```diff
@@ -49,7 +53,7 @@
                 </label>
                 <div class="custom-select">
                     <select class="custom-select_select"
-                            data-placeholder="Select State"
+                            data-placeholder="Select state"
```

Comment:
Should this be downcased? It's uppercased on https://github.com/cfpb/cfgov-refresh/pull/843/files#diff-1f0688a2a043180c12e1b17d5084cf56R52